servermaster/resource/capacity_impl.go

```diff
@@ -51,14 +51,15 @@ func (m *CapRescMgr) Allocate(tasks []*pb.ScheduleTask) (bool, *pb.TaskScheduler
 }
 
 // Update implements RescMgr.Update
-func (m *CapRescMgr) Update(id model.ExecutorID, use model.RescUnit, status model.ExecutorStatus) error {
+func (m *CapRescMgr) Update(id model.ExecutorID, used, reserved model.RescUnit, status model.ExecutorStatus) error {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 	exec, ok := m.executors[id]
 	if !ok {
 		return errors.ErrUnknownExecutorID.GenWithStackByArgs(id)
 	}
-	exec.Used = use
+	exec.Used = used
+	exec.Reserved = reserved
```

Comment:
I forget the meaning of `Reserved`, would you please add some comments in the definition of `ExecutorResource`, about the meaning of `capacity`, `reserved`, `used` and how to use them. 😂

Reply:
ok